server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/SetMailboxesMethodTest.java

```diff
@@ -773,7 +774,7 @@ public void setMailboxesShouldReturnNotUpdatedWhenUnknownMailbox() {
             .body(NAME, equalTo("mailboxesSet"))
             .body(ARGUMENTS + ".notUpdated", hasEntry(equalTo("unknown-update-id01"), Matchers.allOf(
                     hasEntry(equalTo("type"), equalTo("notFound")),
-                    hasEntry(equalTo("description"), equalTo("The mailbox 'unknown-update-id01' was not found")))));
+                    hasEntry(equalTo("description"), containsString("unknown-update-id01")))));
```

Comment:
Why can't it contain the whole error message ? Or at least a greater subset of it ?